smpe/bld/smpe-split.sh

```diff
@@ -104,9 +104,10 @@ list="$list components/api-mediation/api-catalog-services.jar"  # pax01
 list="$list components/api-mediation/discoverable-client.jar"   # pax02
 list="$list components/api-mediation/discovery-service.jar"     # pax03
 list="$list components/api-mediation/gateway-service.jar"       # pax04
-list="$list components/app-server"                              # pax05
-list="$list components/files-api"                               # pax06
-list="$list components/jobs-api"                                # pax07
+list="$list components/api-mediation/caching-service.jar"       # pax05
```

Comment:
I think this might the ptf generation. In the past there were checks to ensure that the correct number of pax segments were used. Have you run the ptf build and tests?

Reply:
I didn't, I didn't have a clear clue when I changed this, cause the build was failing that time. I will try to run the ptf build

Reply:
It looks like from https://wash.zowe.org:8443/view/Zowe/job/zowe-install-packaging/view/change-requests/job/PR-1783/32 this has broken the ptf build. Try undoing it and rerun the build

Reply:
Done